packages/@vuepress/theme-default/styles/index.styl

```diff
@@ -198,4 +207,22 @@ th, td
     .page
       padding-left 0
 
+::-webkit-scrollbar
```

Comment:
What's the reason for these browser prefixes? Are we seeing a bug with this?

Reply:
the side bar is not behaving well on narrow screens.
![image](https://user-images.githubusercontent.com/33315834/81555828-d7635480-93bb-11ea-90c0-39cb6a50dab5.png)
![image](https://user-images.githubusercontent.com/33315834/81556160-70926b00-93bc-11ea-9aed-2cc78ebabdab.png)
If you don't like the change, I can delete it, but I think that at least the new apprearence do not look worse the the origin one.